app/code/Magento/Cms/Model/Wysiwyg/Images/PrepareImage.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\Cms\Model\Wysiwyg\Images;
+
+use Magento\Catalog\Helper\Data;
+use Magento\Cms\Helper\Wysiwyg\Images as ImagesHelper;
+
+class PrepareImage
```

Comment:
Let's make the class name more meaningful
```suggestion
class GetInsertImageContent
```